views/login.ejs

```diff
@@ -16,14 +16,14 @@
 <body>
```

Comment:
Is there a need of login.ejs even after the comments of the last review on this PR?

Reply:
deleted the file bhaiya